portfolio/src/test/java/com/google/sps/CampaignServletTest.java

```diff
@@ -63,36 +63,50 @@
 	SearchGoogleAdsStreamResponse.class, GoogleAdsServiceClient.class, CredentialRetrieval.class})
 public final class CampaignServletTest {
 
+	private static final String SESSION_ID = "sessionId";
+	private static final String CUSTOMER_ID = "1111";
+	private static final String CUSTOMER_ID_NULL = null;
+	private static final String LOGIN_ID = "0000";
+	private static final String REFRESH_TOKEN_VALID_0 = "0000";
+	private static final String REFRESH_TOKEN_VALID_1 = "1111";
+	private static final String REFRESH_TOKEN_INVALID = "8888";
+	private static final String QUERY_VALID = "valid";
+	private static final String QUERY_INVALID = "invalid";
+	private static final String CLIENT_ID = "clientId";
+	private static final String CLIENT_SECRET = "clientSecret";
+	private static final String CLIENT_SECRET_NULL = null;
+	private static final String DEVELOPER_TOKEN = "developerToken";
+	private static final String DEVELOPER_TOKEN_NULL = null;
+
 	@Test
-	public void processErrorJSONTest() {
+	public void processErrorJSON() {
 		MockCampaignsServlet testServlet = new MockCampaignsServlet();
 		String errorJSON = testServlet.processErrorJSON("My error message is here", "300");
 		String expectedJSON = getStringFromFile("expectedJSON/processErrorJSON1.txt");
-		System.out.println(errorJSON);
-		System.out.println(expectedJSON);
 		Assert.assertTrue(equivalentJSON(expectedJSON, errorJSON));  
 	}
 
+	/* The path leads to a String */
 	@Test
-	public void getValueFromJSONTest1() {
+	public void getValueFromJSON1_validFullPath() {
 		MockCampaignsServlet testServlet = new MockCampaignsServlet();
 		String inputString = getStringFromFile("input/getValueJSON1.txt");
 		JSONObject inputObject = new JSONObject(inputString);
 		String value = testServlet.getValueFromJSON(inputObject, "meta.status");
 		Assert.assertEquals("300", value);
 	}
 
-	/* Test to see if an invalid path will throw an error */
 	@Test(expected = Exception.class)
-	public void getValueFromJSONTest2() {
+	public void getValueFromJSON2_invalidPath() {
```

Comment:
if possible test names should describe result i.e. getValueFromJSONShouldErrorOnInvalidPath